Simplified/Book/UI/NYPLBookCellDelegate+Audiobooks.m

```diff
@@ -20,6 +20,8 @@
 #import "NYPLJSON.h"
 #import "NSString+NYPLStringAdditions.h"
 
+static NSInteger AudiobookProgressSavingInterval = 60;
```

Comment:
can you make this a `const` and add NYPL as a prefix?

Reply:
This has been renamed and moved to the `NYPLBookCellDelegate+AudiobookProgressSaving.swift`